frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -804,35 +869,47 @@ mod tests {
 
 	#[test]
 	fn ocw_check_prevent_duplicate() {
+		const OFFCHAIN_REPEAT: BlockNumber = 5;
```

Comment:
This value should come from <mock::Runtime as Config>::OffchainRepeat

Reply:
81ceca174